### ISSUE TITLE
No sound in Shadow Warrior

### ISSUE THREAD

**Issue by XakepSDK** [ValveSoftware/steam-runtime#254]
#### Your system information

* Steam Runtime Version: 0.20200415.0
* Distribution (e.g. Ubuntu 18.04): Arch Linux
* Link to your full system information (Help -> System Information) in a [Gist](https://gist.github.com/): https://gist.github.com/XakepSDK/1a1af2e6aa914ef618ef788f8b4ff0b3
  
* Have you checked for system updates?: [Yes/No] Yes
* Are you using the Steam Linux Runtime compatibility tool?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
No sound in Shadow Warrior

#### Steps for reproducing this issue:

1. Start game
2. Sound is present in intro
3. No sound in game or game menu

**Comment by smcv** [comment]
Does sound work normally when not using the Steam Linux Runtime compatibility tool?

**Comment by XakepSDK** [comment]
Yes, it works. Does SLR ship lib32-openal? I remember i had same issue with non-SLR setup and it was fixed by installing some 32bit sound library

**Comment by smcv** [comment]
The `libopenal1:i386` package in the container is a 32-bit `libopenal.so.1`, which is the same library that some distributions would package as `lib32-openal`. However, as with basically every library in the container runtime, it's a really old version.

In principle, games are meant to be compiled and tested with the ancient versions from SteamRT 1 'scout', but in practice game developers had no practical way to do that until recently, so perhaps Shadow Warrior is assuming you have a newer version available.

The container runtime is a very strict SteamRT 1 'scout' environment where the majority of libraries from your host system are not used, even if they are newer. (That's sort of the point - it makes it more predictable - but it does mean that any game that is incorrectly assuming a newer version than we were guaranteeing will not work as intended.)

**Comment by XakepSDK** [comment]
I think sound worked 2 releases ago. I did not tested previous release, i'm not sure about it.
Is there any way to install older SLR?

**Comment by smcv** [comment]
There are two major components in the SteamLinuxRuntime: pressure-vessel (the program that sets up the container for your game), and the actual container runtime.

For pressure-vessel, you can download older versions from https://repo.steampowered.com/steamrt-images-scout/snapshots/ (the file you want in this case is `pressure-vessel-bin.tar.gz`). Replace the `SteamLinuxRuntime/pressure-vessel/` directory with the result of unpacking the tar file.

For the container, you can also download older versions from https://repo.steampowered.com/steamrt-images-scout/snapshots/ (the file you want is `com.valvesoftware.SteamRuntime.Platform-amd64,i386-scout-runtime.tar.gz`). The production version of SteamLinuxRuntime only has one copy, but you can actually have several of these available at the same time: unpack it into a directory like `SteamLinuxRuntime/scout_0.20200214.0`, so that you have `SteamLinuxRuntime/scout_0.20200214.0/files` and `SteamLinuxRuntime/scout_0.20200214.0/metadata`, then run Steam with the `PRESSURE_VESSEL_WRAP_GUI` environment variable set to 1, and choose the older runtime.

If you report any bugs or test results with this setup, please be very clear about which version you are using.
